app/controllers/transactions/transaction-list.controller.js

```diff
@@ -25,18 +25,28 @@ module.exports = async function showTransactionList (req, res, next) {
     return next(new Error('Invalid search'))
   }
 
+  let noTransactionSearchResults = {
+    total: 0,
+    count: 0,
+    page: 1,
+    results: [],
+    _links: {},
+    filters: {},
+  }
   let result
+
   try {
-    result = await Promise.all([
-      transactionService.search([accountId], filters.result),
-      client.getAllCardTypes()
-    ])
+      result = await Promise.all([
+        (filters.dateRangeState.isInvalidDateRange) ?
```

Comment:
To make it a bit simpler - can we move the tertiary statement out of the Promise.all?